packages/base-driver/README.md

```diff
@@ -12,6 +12,10 @@ Inactive icon: Opacity 50%, no margins and paddings
 
 ## Changelog
 
+### v0.2.0
+
+- Added SSH tunneling support
```

Comment:
```suggestion
- Added SSH tunneling support. 
```

Reply:
A tiny nitpick.

Reply:
I've fixed the tiny nitpick and force-pushed the changes! Sorry for such a trivial issue.
https://github.com/mtxr/vscode-sqltools/pull/1447/commits/dc08c7bc6129711302712649ae25f588380908da